server/datastore/sql/index.ts

```diff
@@ -59,22 +59,47 @@ export class SqlDataStore implements Datastore {
     );
   }
 
-  getPost(id: string): Promise<Post | undefined> {
-    throw new Error('Method not implemented.');
+  async getPost(id: string): Promise<Post | undefined> {
+    // throw new Error('Method not implemented.');
+    return await this.db.get<Post>('SELECT TOP 1 From posts where postId = ?', id);
   }
-  deletePost(id: string): Promise<void> {
-    throw new Error('Method not implemented.');
+
+  async deletePost(id: string): Promise<void> {
+    await this.db.run('Delete FROM posts WHERE id=?', id);
   }
-  createLike(like: Like): Promise<void> {
-    throw new Error('Method not implemented.');
+
+  async createLike(like: Like): Promise<void> {
+    await this.db.run('INSERT INTO Likes(UserId,PostId) VALUES(?,?)', like.userId, like.postId);
```

Comment:
Let's try to be a little more careful with case sensitivity in SQL, it's kind of a big topic, different across different database engines. I always try to do keywords (e.g. `SELECT`, `FROM`... etc) in upper case, and then all identifiers (table and column names) in camelCase.

Reply:
I got you.
Thanks